extensions/vertx-web/deployment/src/main/java/io/quarkus/vertx/web/deployment/VertxWebProcessor.java

```diff
@@ -69,6 +71,15 @@
 
     HttpConfiguration httpConfiguration;
 
+    private static boolean isApolloInClasspath() {
+        try {
+            Class.forName("io.vertx.ext.web.handler.graphql.ApolloWSHandler");
```

Comment:
In general in Quarkus we use an additive approach where we define extensions for additional features, rather than the Spring Boot approach of checking what is on the CP to decide what to enable. It looks like GraphQL will require config and reflection so I really think this should be its own extension.